docs/content/en/contribute/_index.md

```diff
@@ -26,9 +26,25 @@ Also, we encourage you to submit Issues, starting from https://github.com/k8ssan
 CI will run on push to any branch of your forked repository. In order to run CI jobs involving S3 buckets (for Medusa), two secrets need to be set up in the fork organization:
 
 - K8SSANDRA_MEDUSA_BUCKET_NAME: Name of the S3 bucket used to store Medusa backups.
-- K8SSANDRA_MEDUSA_BUCKET_SECRET: Region of the S3 bucket.
-- MEDUSA_SECRET: A valid S3 key/secret pair to access the S3 bucket.
-
+- K8SSANDRA_MEDUSA_BUCKET_REGION: Region of the S3 bucket.
+- K8SSANDRA_MEDUSA_SECRET_S3: A valid Kubernetes secret definition to access the S3 bucket.
+
+For S3, your secret definition _must_ be similar to the one below:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  # The name must be medusa-bucket-key!
+  name: medusa-bucket-key
+type: Opaque
+stringData:
+# The entry must be named medusa_s3_credentials!
+ medusa_s3_credentials: |-
+   [default]
+   aws_access_key_id = <your key id>
+   aws_secret_access_key = <your key secret>
+```
```

Comment:
Since we're adding other storage providers in CI, we may want to give some info on how to set them up as well? wdyt?

Reply:
Good idea, will do that now.

Reply:
Implemented in 100d5ed